test/Microsoft.Framework.PackageManager.FunctionalTests/Microsoft.Framework.PackageManager.FunctionalTests.xproj

```diff
@@ -13,5 +13,8 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```

Comment:
Per @dougbu I'll unrevert this (unvert? devert?). It's something VS adds to test projects to avoid having to rescan them to determine if they are test projects and the sooner we get it in, the less churn we have later :)